blog-server-services/src/impls/rbatis_post_service.rs

```diff
@@ -15,6 +15,7 @@ impl_insert!(BasePost {}, "post");
 pub struct TagDto {
     post_id: i64,
     id: i64,
```

Comment:
u64 🤡

Reply:
changed, also for some methods and other entities